package.json

```diff
@@ -9,6 +9,21 @@
 		"format": "prettier --write",
 		"lint-and-fix": "eslint . --ext .ts --fix"
 	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.{js,jsx,ts,tsx}": [
+			"prettier --write",
+			"eslint --fix",
```

Comment:
If there is any linting error, I think this would fix it but not add it to staging area. Just confirm it once, try committing a file with some lint error. 
You can follow these steps:
```bash
git add <filename>
git commit 
git reset HEAD~1 
```
this will revert back the commit.
If the linting errors are fixed and also being committed, then we will be good to go otherwise remove the `fix`

Reply:
I will check and confirm.